tests/src/traffic/steal.rs

```diff
@@ -185,6 +186,60 @@ mod steal {
         application.assert(&client);
     }
 
+    #[rstest]
+    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+    #[timeout(Duration::from_secs(120))]
+    async fn filter_with_single_client_and_only_matching_requests_by_path(
```

Comment:
This test is called `...only_matching_requests...` but contains a `GET` request that is unmatched.
I don't know if sending that `GET` request somehow verifies that the request was not stolen.

I suggest changing the test's name, removing the `GET` request, and adding a `DELETE` request that doesn't get stolen before sending the one that does. That way we actually verify that an unmatched request is not stolen - because if it were, the local app would exit. There is already a comment for that in line 228, that I think is currently wrong.

Reply:
We verify the response

Reply:
Nice, so we can just change the test name and remove the wrong comment.

Reply:
Yup, done!